lib/jsdom/living/nodes/ChildNode-impl.js

```diff
@@ -1,6 +1,38 @@
 "use strict";
+const { createNode } = require("../node");
 
 class ChildNodeImpl {
+  after(...nodes) {
+    const parent = this.parentNode;
+    if (parent) {
+      let viableNextSibling = this.nextSibling;
+      let idx = viableNextSibling ? nodes.indexOf(viableNextSibling) : -1;
+
+      while (idx !== -1) {
+        viableNextSibling = viableNextSibling.nextSibling;
+        if (!viableNextSibling) {
+          break;
+        }
+        idx = nodes.indexOf(viableNextSibling);
+      }
+      parent.insertBefore(createNode(this._ownerDocument, nodes), viableNextSibling);
+    }
+  }
+
+  before(...nodes) {
```

Comment:
Thanks, I did .before

for .replaceWith I don't understand the [5.](https://dom.spec.whatwg.org/#dom-childnode-replacewith) I started reimplementing the replace algorithm, but I figured out after that it was already done in NodeImpl#replaceChild, so I've just this step to understand